src/main/java/org/broadinstitute/hellbender/utils/IndexUtils.java

```diff
@@ -23,23 +23,63 @@ private IndexUtils(){}
     private static final Logger logger = LogManager.getLogger(IndexUtils.class);
 
     /**
+     * Get the index file associated with a given {@code featureFile}
+     *
+     * @param featureFile The feature file whose index we're trying to get.
+     * @return The index file corresponding to the given {@code featureFile} ; null if it doesn't exist.
+     */
+    public static File getIndexFile(final File featureFile) {
```

Comment:
Don't implement new file-based methods to load the index file. Instead, use the existing methods in htsjdk that have already been generalized to work with non-file inputs such as GCS URIs. Eg., `IndexFactory.loadIndex()`, `Tribble.indexFile(String)`, etc. 

Make sure to include tests with files in GCS.

Reply:
Sounds good.